packages/react-components/react-color-picker-preview/stories/src/ColorPicker/ColorAndSwatchPicker.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { tinycolor } from '@ctrl/tinycolor';
+import { makeStyles, Button, SwatchPicker, EmptySwatch, ColorSwatch, Label } from '@fluentui/react-components';
+import {
+  ColorPicker,
+  ColorSlider,
+  AlphaSlider,
+  ColorPickerProps,
+  ColorArea,
+} from '@fluentui/react-color-picker-preview';
+import type { SwatchPickerOnSelectEventHandler } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  example: {
+    width: '300px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+  },
+  previewColor: {
+    width: '50px',
+    height: '50px',
+    borderRadius: '4px',
+    border: '1px solid #ccc',
+    margin: '10px 0',
+  },
+  button: {
+    marginRight: '8px',
+  },
+  input: {
+    display: 'block',
+    margin: '6px 0',
+  },
+  row: {
+    display: 'flex',
+    gap: '10px',
+    justifyContent: 'space-between',
+  },
+  sliders: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '80%',
+  },
+});
+
+const ITEMS_LIMIT = 8;
+const DEFAULT_COLOR_HSV = tinycolor('#2be700').toHsv();
+
+export const ColorAndSwatchPickerExample = () => {
+  const styles = useStyles();
+  const [color, setColor] = React.useState(DEFAULT_COLOR_HSV);
+  const [selectedValue, setSelectedValue] = React.useState('fff');
+  const [selectedColor, setSelectedColor] = React.useState('#fff');
+
+  const colorFocusTargetRef = React.useRef<HTMLButtonElement>(null);
+  const [colorFocusTarget, setColorFocusTarget] = React.useState<string | null>(null);
+
+  const [items, setItems] = React.useState<Array<{ color: string; value: string; 'aria-label': string }>>([]);
+  const emptyItems = new Array(ITEMS_LIMIT - items.length).fill(null);
+
+  const handleChange: ColorPickerProps['onColorChange'] = (_, data) => {
+    setColor({ ...data.color, a: data.color.a ?? 1 });
+  };
+
+  const handleSelect: SwatchPickerOnSelectEventHandler = (_, data) => {
+    setSelectedValue(data.selectedValue);
+    setSelectedColor(data.selectedSwatch);
+  };
+
+  const handleAddColor = () => {
+    const newColor = tinycolor(color).toRgbString();
+    const newValue = `custom-${newColor} [${items.length - ITEMS_LIMIT}]`;
+
+    setItems([...items, { color: newColor, value: newValue, 'aria-label': newColor }]);
+    setColorFocusTarget(newValue);
+  };
+
+  const resetColors = () => {
+    setItems([]);
+    setColorFocusTarget(selectedValue);
```

Comment:
shouldn't it reset to a default values? :)

```suggestion
    setColorFocusTarget(null);
    setSelectedValue('fff');
    setSelectedColor('#fff');
    setColor(DEFAULT_COLOR_HSV);
```

Reply:
This is part of a11y fix: #32157